### PR TITLE
Add `OrganizationMembership.create_or_update{,!}`

### PR DESCRIPTION
**Pull request by ojab**
Didn't tested because I don't have proper zendesk credentials ._.

### CODE REVIEW

File: spec/live/organization_spec.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def valid_attributes
         end
       end
 
-      it "updates the existing user" do
+      it "updates the existing organization" do
```
Comment (ojab) [inline spec/live/organization_spec.rb:33]: Drive-by typo fix that doesn't warrant a separate PR.

**Comment by ojab** [comment]
…and endpoint is 404 :disappointed: 
Should I create a support ticket for that?

**Comment by nogates** [comment]
> Should I create a support ticket for that?

No! it's all good. PR workflows don't actually have permissions to read repository secretes, unless the PR is created within the repo (which is not the case).

But , I fetched this reference and pushed it to a local branch we use to run spec_live tests, and everything seems to be working [fine](https://github.com/zendesk/zendesk_api_client_rb/runs/4691064130?check_suite_focus=true)! so yeah, merging!

Thanks for the contribution!
